### ISSUE TITLE
BUG: Unable to connect to sql server using ssl with self-signed cert + specify domain

### ISSUE THREAD

**Issue by fpauser** [dbgate/dbgate#305]
Trying to connect to a windows sql server with self signed certificate:

```
Error JSON: {
  "code": "ELOGIN"
}

Connection: {
  "server": "***",
  "engine": "mssql@dbgate-plugin-mssql",
  "user": "***",
  "password": "***",
  "defaultDatabase": "***",
  "unsaved": false,
  "_id": "***",
  "connectionColor": "***",
  "useSshTunnel": false,
  "useSsl": true,
  "sshMode": "userPassword",
  "sshKeyfile": "***",
  "sslRejectUnauthorized": false
}

Platform: linux
```

SSL is enabled, unauthorized access is allowed:
![grafik](https://user-images.githubusercontent.com/7302/176921770-3fc51ece-1e81-4a2b-8bac-ea4d9201297e.png)

From my understanding this should work, or?

**Comment by janproch** [comment]
It should work in newest BETA, there is new option "Trust server certificate", after checking it self-signed certificate should be accepted

**Comment by fpauser** [comment]
Side question: How to specify the domain when connecting? Just `<domain>/<user>` in the user input?

**Comment by janproch** [comment]
Under Windows, you can use integrated or SQL server authentication, then use standard form domain/user
Under mac and linux now it is not possible, I will add this option into next version

**Comment by janproch** [comment]
Support for domain login under linux and mac is added into current BETA. Just fill domain, then NTLM auth will be used

**Comment by fpauser** [comment]
Thanks for your response! Will test tomorrow and report back.

**Comment by fpauser** [comment]
Unfortunately it does not work - and the error message is not of any help.

**Comment by janproch** [comment]
Domain login fixed in 5.1.0

**Comment by fpauser** [comment]
Great - thanks for fixing this!
